src/Tribe/PUE/Checker.php

```diff
@@ -1614,6 +1651,15 @@ public function request_info( $query_args = [] ) {
 					'Accept' => 'application/json',
 				],
 			];
+
+			/**
+			 * Filters the options used in the wp_remote_get() call to request plugin info.
+			 *
+			 * @since TBD
```

Comment:
```suggestion
			 * @since 5.2.0
```
Just assume if they already exist they were added back in 5.2.0 - most of them were and we'll be ~98% accurate 😉